src/main/kotlin/uk/gov/dluhc/printapi/client/ElectoralRegistrationOfficeManagementApiClient.kt

```diff
@@ -21,43 +22,56 @@ class ElectoralRegistrationOfficeManagementApiClient(
     private val eroMapper: EroManagementApiEroDtoMapper
 ) {
 
+    /**
+     * Calls the `ero-management-api` to return a [ElectoralRegistrationOfficeResponse] for the specified eroId.
+     *
+     * @param eroId the ID of the ERO to return
+     * @return a [ElectoralRegistrationOfficeResponse] for the ERO
+     * @throws [ElectoralRegistrationOfficeManagementApiException] concrete implementation if the API returns an error
+     */
+    fun getElectoralRegistrationOffice(eroId: String): ElectoralRegistrationOfficeResponse =
+        eroManagementWebClient
+            .get()
+            .uri("/eros/{eroId}", eroId)
+            .retrieve()
+            .bodyToMono(ElectoralRegistrationOfficeResponse::class.java)
+            .onErrorResume { ex -> handleException(ex, "eroId = $eroId") }
+            .block()!!
+
     /**
      * Calls the `ero-management-api` to return a [EroManagementApiEroDto] for the specified gssCode.
      *
      * @param gssCode the gssCode of the localAuthority for the ERO to be returned
      * @return an [EroManagementApiEroDto] for the ERO
      * @throws [ElectoralRegistrationOfficeManagementApiException] concrete implementation if the API returns an error
      */
-    fun getElectoralRegistrationOffice(gssCode: String): EroManagementApiEroDto {
+    fun getElectoralRegistrationOffices(gssCode: String): EroManagementApiEroDto {
```

Comment:
@Neil-Massey , since this will always return a single ero if successful, the naming confused me actually. 

Maybe something like this `getElectoralRegistrationOfficeByGssCode` is more clear?

Reply:
I think this is how the diff is displayed, this method name has not been changed